.jenkins/job_beam_PreCommit_Website_Stage.groovy

```diff
@@ -0,0 +1,66 @@
+import common_job_properties
```

Comment:
Not feasible without doing ugly things inside the seed job. Seed job builds by cloning this repo and running all jobs inside the .jenkins folder -- we could hypothetically also manually clone the website repo, have its groovy jobs reference (essentially) ../../beam/.jenkins/common... but that seems to me to be a fairly brittle and undesirable solution.